### ISSUE TITLE
Suggestion: include required libs

### ISSUE THREAD

**Issue by LuciousLisa** [Vazkii/CMPDL#25]
Considering the dependency on the gson and zip4j libraries, could you please include those?  
It would be very helpful if users don't have to figure out how to get them before following the steps as described in the README's usage instructions, particularly when the distro maintainers haven't packaged the libraries in question, such as Debian.

**Comment by frenchiveruti** [comment]
I agree with this, I had to run all over the place search for the libs haha.

**Comment by nhinds** [comment]
It looks like the .jar on the [Releases](https://github.com/Vazkii/CMPDL/releases) page has gson and zip4j inlined, at least in the latest version. This is probably as simple as updating the README to say you don't need to get gson/zip4j yourself if you want to run the program from the .jar.
